lib/whitehall_importer/integrity_checker/body_check.rb

```diff
@@ -0,0 +1,29 @@
+module WhitehallImporter
+  class IntegrityChecker::BodyCheck
```

Comment:
I wonder if this should be BodyTextCheck? I'm a bit confused as we seem to mix up whether we describe the field as body or body text and it'd be simpler if we used the same phrasing consistently.

Reply:
I agree, name changed to `BodyTextCheck`.